src/coreclr/jit/lir.cpp

```diff
@@ -1164,14 +1164,26 @@ bool LIR::Range::TryGetUse(GenTree* node, Use* use)
 // occurring before uses).
 //
 // Arguments:
-//    root        - The root of the dataflow tree.
-//    isClosed    - An output parameter that is set to true if the returned
-//                  range contains only nodes in the dataflow tree and false
-//                  otherwise.
+//    markFlagsOperands - Whether the dataflow algorithm should also try to
+//                        mark operands that are defining flags. For example,
+//                        GT_JCC implicitly consumes the flags defined by the
+//                        previous node in linear order. If this argument is
+//                        true, then the algorithm will also include the flags
+//                        definition (which is e.g. a CMP node) in the returned
+//                        range.
+//                        This works on a best-effort basis; the user should not
+//                        rely on all flags defs to be found.
+//    root              - The root of the dataflow tree.
+//    isClosed          - An output parameter that is set to true if the returned
+//                        range contains only nodes in the dataflow tree and false
+//                        otherwise.
```

Comment:
While you're here, can you fix this? `root` doesn't match the declaration. It's missing `markCount` and `sideEffects`.